### DISCUSSION TITLE
Error when install Match3-Sample

### DISCUSSION THREAD

**Discussion by nghelam2020** [ChebanovDD/MatchSweets#10] [Q&A] [answered]
Hi everyone,

I have installed Match3-Sample as in instruction. However, Unity show error "DOTween.Sequence() does not contain WithCancellation()".

 await itemsSequence.SetEase(Ease.Flash).WithCancellation(cancellationToken);

Please help me fix this error. Thank you!

**Comment by ChebanovDD** [comment, marked as answer]
Hello, there.

Looks like the DOTween package wasn't installed. Check it out in the Package Manager.

In addition, I've noticed, that the DOTween package is no longer available in the OpenUPM. This may cause the issue. Try to install it manually.

P.S. When the DOTWeen is installed manually. You have to define `UNITASK_DOTWEEN_SUPPORT` to enable it for UniTask.

**Reply by nghelam2020** [reply]
Thank you!

**Comment by guillempd** [comment]
I'm having this same issue. Installing DOTween manually from the asset store doesn't solve it either for me. Any suggestions?

**Reply by ChebanovDD** [reply]
Could you describe the installation algorithm step by step? Did you just clone the repo or did you install the SDK through the package manager? If through the PM, how did you install the sample?

**Reply by nghelam2020** [reply]
For me, 

1. Create new 2D project
2. Install UniTask and Match3-SDK via Package Manager
3. Import DOTween using Asset menu
4. Once done, open the script in Visual Studio. For example, in the ItemsHideJob.cs file, the error occurs in the ExecuteAsync() method and in the statement with the WithCancellation() function call.
 var itemsSequence = DOTween.Sequence();

await itemsSequence.WithCancellation(cancellationToken);

Thanks you!

**Reply by guillempd** [reply]
I tried two approaches and neither of them worked.

The first approach was to clone the whole repo and open the samples/Unity.Match3 folder as Unity project, I followed this steps:
1. Clone the repo.
2. Open the Unity.Match3 sample project, this already gives somes errors due to DOTween not being available through OpenUPM:
![before-installing-dotween](https://user-images.githubusercontent.com/15831136/205894842-30c813e7-b545-4dd0-aacc-a057be27a6f8.png)
3. Install DOTween manually from package manager (asset store), then gives the following errors where I am stuck:
![after-installing-dotween](https://user-images.githubusercontent.com/15831136/205895350-ea851a82-5130-4023-8d1f-42d05afbec15.png)

Following the approach that @nghelam2020 just commented gives the sames errors as in the last screenshot.

**Reply by ChebanovDD** [reply]
Thanks for the detailed info.
So, when DOTWeen is installed manually. You have to define `UNITASK_DOTWEEN_SUPPORT` to enable it for UniTask.

![image](https://user-images.githubusercontent.com/28132516/206328311-95c7203e-7b92-4275-ba1f-a7a13ff558cd.png)

**Reply by guillempd** [reply]
Great, that was the missing piece, now it works perfectly! Thanks

**Comment by 123SourLemon123** [comment]
Hi Dima, 
I installed Unity 6 and when I try to Install project via Package manager > + > git url > I get an error: 

`[Package Manager Window] Error adding package: https://github.com/LibraStack/Match3-SDK.git?path=src/Match3-SDK/Assets/Match3.
Unable to add package [https://github.com/LibraStack/Match3-SDK.git?path=src/Match3-SDK/Assets/Match3]:
  Cannot checkout repository [https://github.com/LibraStack/Match3-SDK.git] on target path [src/Match3-SDK/Assets/Match3]:
    Error when executing git command. error: pathspec 'src/Match3-SDK/Assets/Match3' did not match any file(s) known to git

UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()
`
or for v0.1.3

`[Package Manager Window] Error adding package: ttps://github.com/ChebanovDD/Match3-SDK.git?path=src/Match3.UnityPackage/Assets/Plugins/Match3#v0.1.3.
Unable to add package [ttps://github.com/ChebanovDD/Match3-SDK.git?path=src/Match3.UnityPackage/Assets/Plugins/Match3#v0.1.3]:
  Package name 'ttps://github.com/ChebanovDD/Match3-SDK.git?path=src/Match3.UnityPackage/Assets/Plugins/Match3#v0.1.3' is invalid.
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()`

How it is possible to install via git url?
